tests/test_webframe.py

```diff
@@ -65,6 +65,14 @@ def test_ico(self) -> None:
         self.assertEqual(len(extra_headers), 1)
         self.assertEqual(extra_headers[0][0], "Last-Modified")
 
+    def test_svg(self) -> None:
+        """Tests the svg case."""
+        content, content_type, extra_headers = webframe.handle_static("/favicon.svg")
+        self.assertTrue(len(content))
+        self.assertEqual(content_type, "image/svg+xml")
+        self.assertEqual(len(extra_headers), 1)
```

Comment:
Ez lehetne assertGreater is, nem? Nem néztem, de ha a keddi tömörítős commit felett vagy, akkor valszeg lesz ott más header is, ami nem probléma, gondolom az assert szándéka, hogy lássuk, lehet-e biztonságosan elérni a 0. elemet.

Reply:
A pontosan felette található `.ico` tesztesetet másoltam le.

Reply:
Most megnéztem a forrásban és az a magyarázat, hogy a `handle_static()` a webframe szintjén nem használ tömörítést. A tömörítést a wsgi rakja rá a `send_response()` hívásban a `handle_static()` visszatérési értékére. Azaz ebben a tesztben nincs ilyen header és ez rendben van így..